src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/SQLitePersistor.java

```diff
@@ -64,7 +64,7 @@ synchronized void createTable(String tableName, List<Field<?>> columns) {
         .column(DSL.field(getPrimaryKeyColumnName(tableName) + PRIMARY_KEY_AUTOINCREMENT_POSTFIX))
         .columns(columns);
 
-    LOG.info("ruizhen: table created: {}", constraintStep.toString());
+    LOG.debug("table created: {}", constraintStep.toString());
```

Comment:
We start every log message inside the RCA package as `RCA:` to distinguish the log from the Reader. can we do the same here ?